versions.json

```diff
@@ -4,5 +4,5 @@
   "geth": "v1.13.4",
   "nvm": "v20.9.0",
   "slither": "0.10.0",
-  "kontrol": "0.1.127"
+  "kontrol": "0.1.178"
```

Comment:
Does this version include the JUint XML reports from https://github.com/runtimeverification/kontrol/issues/329? Would be great to include that in this PR if we can. Relevant CircleCI docs [here](https://circleci.com/docs/collect-test-data/), and we currently use that for our Go tests if you want to see the implementation in the existing CI file

Reply:
That Kontrol version doesn't include it, but we can add it without a problem! My only question is, in light of offloading computation, would it still make sense to add the reports to this PR? Or would it be best to wait? Not sure how the CI landscape will change

Reply:
We can modify which tests your machines will execute from this PR so that resources will not be exhausted. This would allow us to merge this PR first, having the final code in place, and not have to wait to have the "offload compute" PR merged first. We can do it either way, just wanted to note that we can set it to execute all tests later

Reply:
Ok perfect, I'm good with that approach. So the only remaining blockers for merge here would be changing the set of tests to run, and adding the JUnit XML reports for those?

Reply:
An update on this, we're pending on one PR to support generating the JUint XML reports with tests in nested dirs. Tomorrow should be ready, and after that, everything will be ready to go!

Reply:
The last commits should contain everything related to this conversation